utils/migration_tools/backup_neptune/README.md

```diff
@@ -0,0 +1,97 @@
+# Back up run metadata from Neptune
+
+You can use this script to download run metadata from Neptune to your system. Note that this script is currently in beta, and we welcome your feedback and contributions to help improve it.
+
+## Prerequisites
+
+Before using this script, make sure you have
+1. the Neptune environment variables set up. For instructions, see the [documentation](https://docs.neptune.ai/setup/setting_credentials/).
+2. `tqdm` installed using `pip install -U tqdm`
+
+## Instructions
+
+The script will generate run logs in the same folder as `bulk_download_metadata.py`. You can modify this location in the `logging.basicConfig()` function.
+
+To use the script, follow these steps:
+
+1. Run `bulk_download_metadata.py`.
+1. Enter the download path where you want the metadata to be downloaded.
+1. Indicate if you want remotely tracked artifacts to be downloaded.
+1. All eligible projects will be displayed in the console. Enter the projects you want to download the run metadata from.
```

Comment:
One last thing. Since we offer the steps here, let's make it crystal clear how to enter the project name.

```suggestion
1. All projects you have access to are displayed in the console. Enter the projects you want to download the run metadata from.

    Example: `ml-team/classification ml-team/image-segmentation`
```

Reply:
Projects are entered as comma-separated-values, without spaces
Example: `ml-team/classification,ml-team/image-segmentation`